docs/ingestion/input-sources.md

```diff
@@ -1141,22 +1141,122 @@ To use the Delta Lake input source, load the extension [`druid-deltalake-extensi
+|----------|-----------------------------------------------------------------------------------------------------------------------------------------------------------------|----------|
+| type     | Set this value to `or`.                                                                                                                                         | yes      |
+| filters  | List of Delta filter predicates that get evaluated using logical OR where only one condition needs to be true. `or` filter requires two filter predicates.      | yes      |
+
+`not` filter:
+
+| Property | Description                                                                                                   | Required |
+|----------|---------------------------------------------------------------------------------------------------------------|----------|
+| type     | Set this value to `not`.                                                                                      | yes      |
+| filter   | The Delta filter predicate that gets evaluated using logical NOT. `not` filter requires one filter predicate. | yes      |
+
+`=` filter:
+
+| Property | Description                              | Required |
+|----------|------------------------------------------|----------|
+| type     | Set this value to `=`.                   | yes      |
+| column   | The table column to apply the filter on. | yes      |
+| value    | The value to use in the filter.          | yes      |
+
+`>` filter:
+
+| Property | Description                              | Required |
+|----------|------------------------------------------|----------|
+| type     | Set this value to `>`.                   | yes      |
+| column   | The table column to apply the filter on. | yes      |
+| value    | The value to use in the filter.          | yes      |
+
+`>=` filter:
+
+| Property | Description                              | Required |
+|----------|------------------------------------------|----------|
+| type     | Set this value to `>=`.                  | yes      |
+| column   | The table column to apply the filter on. | yes      |
+| value    | The value to use in the filter.          | yes      |
+
+`<` filter:
+
+| Property | Description                              | Required |
+|----------|------------------------------------------|----------|
+| type     | Set this value to `<`.                   | Yes      |
+| column   | The table column to apply the filter on. | Yes      |
+| value    | The value to use in the filter.          | Yes      |
+
+`<=` filter:
+
+| Property | Description                              | Required |
+|----------|------------------------------------------|----------|
+| type     | Set this value to `<=`.                  | yes      |
+| column   | The table column to apply the filter on. | yes      |
+| value    | The value to use in the filter.          | yes      |
```

Comment:
There is some redundancy in the docs. Perhaps, we can merge all these filters into a single table. 
```suggestion
| Property | Description                              | Required |
|----------|------------------------------------------|----------|
| type     | Set this value to '=', '>=', ....`<=`.                  | yes      |
| column   | The table column to apply the filter on. | yes      |
| value    | The value to use in the filter.          | yes      |
```